lib/menu.rb

```diff
@@ -0,0 +1,13 @@
+class Menu
+
+  attr_reader :display_menu
+
+  def initialize
+    @display_menu = [["Pounded Yam", 15],
```

Comment:
Good, although I really think a hash would be appropriate here instead of an array as then you wouldn't have those magic numbers in the customer class.